### DISCUSSION TITLE
Style <Select> selected(active) item

### DISCUSSION THREAD

**Discussion by vkatsou** [mantinedev/mantine#3780] [Q&A] [answered]
<img width="494" alt="Screenshot 2023-03-15 at 11 27 22 AM" src="https://user-images.githubusercontent.com/11488579/225267753-5ee01b3a-4ae0-4ec6-a07c-09bdb564c946.png">
Is it possible to style the <Select> active item.
I've tried using the styles.item prop with '$:active', '$:selected', '$:focus', '$:hover' but nothing seems to do the trick.

**Comment by yverby** [comment, marked as answer]
Use `&[data-selected]` for `item` selector

**Reply by vkatsou** [reply]
Thanks this was it!

**Comment by yverby** [comment]
You can find the rest of `item` states here
https://github.com/mantinedev/mantine/blob/master/src/mantine-core/src/Select/SelectItems/SelectItems.styles.ts
